app/views/shared/_recently_updated_from_rummager.html.erb

```diff
@@ -0,0 +1,21 @@
+<% extra_class ||= '' %>
```

Comment:
Question: what exactly is `extra_class` and in what situations is it provided? Is it possible to make this variable name a little clearer?